tests/phpunit/FileSystem/SourceFileCollector/SourceFileCollectorTest.php

```diff
@@ -36,32 +36,38 @@
 namespace Infection\Tests\FileSystem\SourceFileCollector;
 
 use Infection\FileSystem\SourceFileCollector;
+use function iterator_to_array;
 use function ksort;
 use PHPUnit\Framework\Attributes\CoversClass;
 use PHPUnit\Framework\Attributes\DataProvider;
 use PHPUnit\Framework\TestCase;
-use function Pipeline\take;
 use SplFileInfo;
 use Symfony\Component\Filesystem\Path;
 
 #[CoversClass(SourceFileCollector::class)]
 final class SourceFileCollectorTest extends TestCase
 {
-    private const FIXTURES = __DIR__ . '/Fixtures';
+    private const FIXTURES_ROOT = __DIR__ . '/Fixtures';
 
     /**
      * @param string[] $sourceDirectories
      * @param string[] $excludedFilesOrDirectories
      * @param list<string> $expectedList
      */
     #[DataProvider('sourceFilesProvider')]
-    public function test_it_can_collect_files(array $sourceDirectories, array $excludedFilesOrDirectories, array $expectedList): void
-    {
-        $files = (new SourceFileCollector())->collectFiles($sourceDirectories, $excludedFilesOrDirectories);
+    public function test_it_can_collect_files(
+        array $sourceDirectories,
+        array $excludedFilesOrDirectories,
+        array $expectedList,
+    ): void {
+        $files = (new SourceFileCollector())->collectFiles(
+            $sourceDirectories,
+            $excludedFilesOrDirectories,
+        );
 
         self::assertIsEqualCanonicalizing(
             $expectedList,
-            take($files)->toAssoc(),
+            iterator_to_array($files),
```

Comment:
When I see `iterator_to_array()` I have to recall or look up what the the default behavior, so it is almost always a waste of a reader's time; with `take($files)->toAssoc()` there's no such problem - it does what it says. No second guessing, no extra time spend looking up what is what.

Further, calling `iterator_to_array` the wrong way could lead to some intense debugging, so it is very important that the second argument of  `iterator_to_array` has the correct value. Once again, with `take($files)->toAssoc()` or `take($files)->toList()` there's not confusion whatsoever.

Reply:
Tbh it's a case where I really prefer iter's `toArrayWithKeys()` which is both more describe and direct. But I also don't really to introduce yet another lib for this sort of things